### ISSUE TITLE
Visually style links as links

### ISSUE THREAD

**Issue by marcusrelacion** [AIGAdc/dcdesignweek.org-11ty#7]
Several links on the DCDW 2021 website are styled as buttons which may cause issues for:
- Keyboard users who expect to be able to interact with buttons using space (links only respond to enter)
- Voice command users who attempt to trigger a button by saying, “Click explore all events button.” If the button is actually a link pretending to be a button, they will fail to trigger it.
- Screen reader users who are getting help from a friend. If a friend says, “Click the explore all events button,” and the screen reader user searches their elements list (a common method of navigation where a list of all interactive buttons on a page are listed), they will fail to find that button as it is actually coded as a link.
This also includes consistency issues related to chevron usage
